src/script.js

```diff
@@ -225,29 +238,32 @@ const updateReminders = function () {
 				node.getAttribute("name")
 			);
 
-			if (excludingMe.length > 0) {
-				const name = excludingMe[0].getAttribute("name").toUpperCase();
-				const first = name.charCodeAt(0);
-				const targetElement = email.querySelector(".oZ-x3");
-			
-				if (targetElement && targetElement.getElementsByClassName(AVATAR_CLASS).length == 0) {
-					const avatarElement = document.createElement("div");
-					avatarElement.className = AVATAR_CLASS;
-					if (first >= 65 && first <= 90) {
-						avatarElement.style.background = "#" + nameColors[first - 65];
-					} else {
-						avatarElement.style.background = "#000000";
-						// Some unicode characters are not affected by 'color: white', hence this alternative
-						avatarElement.style.color = "transparent";
-						avatarElement.style.textShadow = "0 0 0 #ffffff";
-					}
-
-					avatarElement.innerText = name[0];
-					targetElement.appendChild(avatarElement);
+			let firstParticipant = participants[0];
+			if (excludingMe.length > 0) { // if there are others in the participant, use one of their initials instead
+				firstParticipant = excludingMe[0];
+			}
+
+			const name = firstParticipant.getAttribute("name");
+			const first = (name && name.toUpperCase().charCodeAt(0)) || 65;
+			const targetElement = email.querySelector(".oZ-x3");
+
+			if (targetElement && targetElement.getElementsByClassName(AVATAR_CLASS).length == 0) {
+				const avatarElement = document.createElement("div");
+				avatarElement.className = AVATAR_CLASS;
+				if (first >= 65 && first <= 90) {
+					avatarElement.style.background = "#" + nameColors[first - 65];
+				} else {
+					avatarElement.style.background = "#000000";
+					// Some unicode characters are not affected by 'color: white', hence this alternative
+					avatarElement.style.color = "transparent";
+					avatarElement.style.textShadow = "0 0 0 #ffffff";
 				}
-			
-				email.classList.add(AVATAR_EMAIL_CLASS);
+
+				avatarElement.innerText = (name && name[0]) || "-";
```

Comment:
What was the motivation behind this line, as well the similar change online 247?

Was there an issue you were running into? Is there some weird scenario where a participant has no name at all?

Reply:
The code prior to the change would only execute this code if there were participants other than 'me'. so it would never run into an issue where there were no users. I added this as a precaution. *In case* there are no participants for some weird reason, it would at least display something. I typically use '-' in UIs where there is 'no value'.